packages/components/src/Stack.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import { Box } from './Box'
+
+export const merge = (...objs) => {
+  return objs.reduce(function(merged, currentValue) {
+    return mergeTwo(merged, currentValue)
```

Comment:
possibly jumping in w/o enough context, but where is `mergeTwo` coming from in this case?